### ISSUE TITLE
1.0.0-alpha0 parent/child query parameters not format()ted correctly

### ISSUE THREAD

**Issue by christopherthielen** [angular-ui/ui-router#2504]
https://plnkr.co/edit/eP09lKTynJjblgTyprvZ?p=preview

Failing unit test: 
```
 it("should handle additional query parameters from concated matchers", function() {
      var m = new UrlMatcher('/parent?qParent');
      var m2 = m.append(new UrlMatcher('/child?qChild'));
      var formatted = m2.format({ qParent: 'parent', qChild: 'child' });

      expect (formatted).toBe('/parent/child?qParent=parent&qChild=child');
    })```

**Comment by pavel-blagodov** [comment]
Could you tell me the approximate date of 1.0.0-alpha.1release?

**Comment by christopherthielen** [comment]
Could be as soon as next week. Are you looking for any fix to alpha0 in particular?

**Comment by pavel-blagodov** [comment]
Yes, I'm. It would be great if this issue will be fixed asap.

**Comment by christopherthielen** [comment]
Fixed in 86e07efcda78f70079e0ccce9e151516b6cdc784
